common/src/commonMain/kotlin/entity/DiscordMessage.kt

```diff
@@ -371,6 +371,7 @@ public data class DiscordPartialMessage(
     @SerialName("referenced_message")
     val referencedMessage: Optional<DiscordMessage?> = Optional.Missing(),
     val interaction: Optional<DiscordMessageInteraction> = Optional.Missing(),
+    val thread: Optional<DiscordChannel> = Optional.Missing(),
```

Comment:
why in `DiscordPartialMessage`?

Reply:
IDK do we have a description how partial a partial message is?

Reply:
> IDK do we have a description how partial a partial message is?

not that i'm aware of. i was more wondering about why it isn't in `DiscordMessage`

Reply:
That's a really good question

Reply:
It is ;) https://github.com/kordlib/kord/blob/1637d259c154aa8d28cbca21b8cadb39e9725d6f/common/src/commonMain/kotlin/entity/DiscordMessage.kt#L238

Reply:
oops xD